spec/db.spec.ts

```diff
@@ -181,6 +185,86 @@ describe('db', () => {
             });
           }
 
+          describe('given ssh-agent is not running', () => {
+            it('should fail to connect into server using ssh agent', () => {
+              const serverInfo: ServerConfig = {
+                ...config[dbAdapter],
+                name: dbAdapter,
+                adapter: dbAdapter,
+              };
+
+              serverInfo.ssh = {
+                host: sshHost,
+                port: 2222,
+                user: 'sqlectron',
+                useAgent: true,
+              };
+
+              const serverSession = db.createServer(serverInfo);
+              const dbConn = serverSession.createConnection(database);
+
+              return expect(assertSSHConnection(dbConn)).to.be.rejectedWith(
+                'not set SSH_AUTH_SOCK env variable',
+              );
+            });
+          });
+
+          describe('given ssh-agent is running', () => {
+            const agentSocket = path.join(__dirname, 'ssh_files/ssh-agent.socket');
```

Comment:
Personal preference would be on this file being placed in the `os.tmpdir()` directory to avoid it falling into the git tree but this is also fine.

Reply:
done